src/ecs/EntityPool.cpp

```diff
@@ -1,89 +1,73 @@
-#pragma once
-
-#include "EntityPool.h"
 #include "utils/Assert.h"
+#include <ecs/EntityPool.h>
 
-#define TEMPLATE_E template<class E>
-#define POOL_E EntityPool<E>
+#define POOL EntityPool
```

Comment:
I think that this macro is pointless. It doesn't shortens that much to be relevant.